src/SFA.DAS.ApprenticeCommitments/Data/Models/CommitmentStatement.cs

```diff
@@ -35,13 +35,19 @@ private CommitmentStatement()
         public DateTime CommitmentsApprovedOn { get; private set; }
         public Apprenticeship Apprenticeship { get; internal set; } = null!;
 
-        public bool? TrainingProviderCorrect { get; private set; }
         public bool? EmployerCorrect { get; private set; }
-        public bool? RolesAndResponsibilitiesCorrect { get; private set; }
+        public bool? TrainingProviderCorrect { get; private set; }
         public bool? ApprenticeshipDetailsCorrect { get; private set; }
+
+        public bool? RolesAndResponsibilitiesCorrect { get; private set; }
         public bool? HowApprenticeshipDeliveredCorrect { get; private set; }
         public bool ApprenticeshipConfirmed => ConfirmedOn.HasValue;
 
+        public bool DisplayChangeNotification =>
+            EmployerCorrect == null
+            || TrainingProviderCorrect == null
+            || ApprenticeshipDetailsCorrect == null;
+
```

Comment:
Took me a while to work out what would happen here, but it makes sense. Minor point should we call it  DisplayChangeNotification? (as that logic belongs to the Aprenticeship) Would it be better to call it SectionNeedToBeConfirmed or something better name